code/style.css

```diff
@@ -1,7 +1,387 @@
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+/* a filter that darkens the video */
+.hero::after { 
+  width: 100%;
+  height: 100%;
+  content: '';
+  z-index: 10;
+  position: absolute;
+  top: 0;
+  left: 0;
+  background: rgba(0,0,0,0.5);
+}
+
+.hero video {
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  top: 0;
+  left: 0;
+  z-index: 5;
+  object-fit: cover;
+}
+
+.hero .caption {
+  position: relative;
+  z-index: 13;
+  text-align: center;
+  color: white;
+  width: 90%
+}
+
+.caption h1 {
+  text-transform: uppercase;
+  font-size: 3em;
+  font-family: 'Oswald', sans-serif;
+}
+
+.caption p {
+  font-size: 1.5em;
+}
+
+.icons {
+  text-align: center;
+  color: orange;
+  font-size: 2em;
+}
+
+/* rain fun facts */ 
+.fun-facts{
+  width: 80%;
+  border-radius: 20px;
+  margin: 15px auto;
+  color: #e7e0c9; 
+  line-height: 1.5;
+  text-align: center;
+  background-color: rgba(255, 255, 255, .15);
+  text-shadow: -1px 1px solid black;  
+}
+
+.fun-facts p {
+    font-size: 1.5em;
+}
+
+.fun-facts ul  {
+  list-style-type: none;
+  padding-inline-start: 0;
+  padding: 10px;
+  font-size: 1.2em;
+}
+
+/* FORMS */
+/* .form is in the "sign in page" */
+
+.form {
+  text-align: center;
+  width: 90%;
+  padding: 20px auto;
+  line-height: 1.6em;
+  margin: 20px 20px;
+  background-color: #22577A;
+  color: #e7e0c9;
+  text-shadow: -1px 1px black;
+  border: 2px solid orange;
+  border-radius: 20px;
+  padding: 10px 5px;
+  cursor: pointer;
+}
+
+/* Title in sign in window */
+.welcome p {
+  font-family: 'Oswald', sans-serif;
+  color: white;
+  margin: 30px auto;
+  text-align: center;
+}
+/* sign up title */
+h3 {
+  text-align: center;
+  border-bottom: 2px solid #e7e0c9;
+}
+
+.signup-box {
+  margin: auto;
+  border: 2px solid orange;
+  border-radius: 20px;
+  width: 80%;
+  padding: 10px 5px;
+  background-color: rgba(255, 255, 255, .15);
+  color: #e7e0c9;
+  text-shadow: -1px 1px black;
+}
+
+input, select, button {
+  background-color: #e7e0c9;
+  cursor: pointer; 
+}
+
+input:focus {
+  background-color: lightgray;
+  outline: 2px solid orange;
+}
+
+.signup-box input, 
+.terms, 
+.age, 
+.gender, 
+.rain-mail {
+  padding: 10px 0;
+  font-size: 0.8em;
+  text-align: center;
+}
+
+.gender-title p {
+  margin-bottom: 10px;
+  font-size: 1.4em;
+}
+
+.radio-title, .checkbox-title, .bold {
+  font-weight: bold;
+}
+
+.form-box, 
+.signup-button {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+/* both buttons */
+button {
+  border: 2px solid orange;
+  box-shadow: 2px 2px 2px black;
+  margin-bottom: 10px;
+  border-radius: 20px;
+}
+
+button:focus {
+  box-shadow: none;
+}
+
+/* message when user registers */
+.greeting {
+  width: 150px;
+  height: 100px;
+  color: #e7e0c9;
+   text-align: center;
+   font-family: 'Oswald' , sans-serif;
+  margin: auto;
+  padding-top: 20px;
+  font-size: 0.88em;  
+}
+
+/* accordion */
```

Comment:
I really liked the accordion, nice with the scrollbar

Reply:
Thank you very much!